From c8a7455058d4612759b8be77a66a738d8f2e50ea Mon Sep 17 00:00:00 2001
From: Georg Lehmann <dadschoorse@gmail.com>
Date: Mon, 12 Apr 2021 16:55:52 +0200
Subject: [PATCH] 3 space tab for mesa

---
 pluma/pluma-window.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pluma/pluma-window.c b/pluma/pluma-window.c
index 1ca50ec..23f16a6 100644
--- a/pluma/pluma-window.c
+++ b/pluma/pluma-window.c
@@ -1916,6 +1916,7 @@ fill_tab_width_combo (PlumaWindow *window)
 {
     static TabWidthDefinition defs[] = {
         {"2", 2},
+        {"3", 3},
         {"4", 4},
         {"8", 8},
         {"", 0}, /* custom size */
-- 
2.34.1

